designs/administrator/olm/catalogsource-as-providertype/index.md

```diff
@@ -0,0 +1,25 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Custom CatalogSource as OperatorHub ‘Provider Type’ filter facet
+
+When a custom CatalogSource is added today, it appears in the “Custom” facet in the ‘Provider Type’ filter in OperatorHub. Customers are interested in having these CatalogSources show up as their own facet instead of being lumped under “Custom.”
```

Comment:
Has anyone thought about how this will affect the developer catalog?  Tiles display "provided by %provider type%".  I'm guessing we should check out how this is handled if this is a long string.

<img width="285" alt="Screen Shot 2020-03-29 at 2 06 03 PM" src="https://user-images.githubusercontent.com/12504403/77856650-88c07900-71c6-11ea-8af5-d159b73aaca3.png">

Reply:
I believe this is actually "provided by %provider%" (at least in the admin operatorhub) so it would not be affected by this change to the provider type. I think that is also the case in the dev catalog?

![provider tile](https://user-images.githubusercontent.com/54860125/77940126-38105500-7286-11ea-8812-09e7d2af3732.png)

Reply:
Tried it for the heck of it, long names/strings look like this in the Developer Catalog:
![image](https://user-images.githubusercontent.com/9122899/77941008-7ce8bb80-7287-11ea-88af-1011ea3f4fab.png)